gradle-conjure/src/test/groovy/com/palantir/gradle/conjure/ConjurePublishTypeScriptTest.groovy

```diff
@@ -119,14 +119,14 @@ class ConjurePublishTypeScriptTest extends IntegrationSpec {
         file('api/api-typescript/src/index.js').text.contains('export * from "./api";')
     }
 
-    def 'compileConjureTypeScript is up-to-date when run for the second time'() {
+    def 'compileTypeScript is up-to-date when run for the second time'() {
```

Comment:
`compileConjureTypeScript` is already tested in `ConjurePluginTest`. Given the structure of these tests, I assume this was originally intended to test `compileTypeScript`.